<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/foto12.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/foto11.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 222, 124);</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>341</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagens/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_perfil">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>20</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    font-size: 30px;
    border-radius: 40px;
}

QPushButton:hover {
    background-color: rgb(255, 255, 255);
}

QPushButton[objectName=&quot;perfilButton&quot;] {
    qproperty-toolTip: &quot;Perfil&quot;;
}

QToolTip {
    font-family: 'SansSerif';
    font-size: 10pt;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../imagens/perfil.png</normaloff>../imagens/perfil.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_voltar">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>70</y>
       <width>139</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
           
	background-color: rgb(111, 125, 255);
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
           background-color: rgb(85, 170, 255); 
        }</string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sair">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>20</y>
       <width>139</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
           
	background-color: rgb(111, 125, 255);
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
           background-color: rgb(85, 170, 255); 
        }</string>
     </property>
     <property name="text">
      <string>SAIR</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>610</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
           
	background-color: rgb(111, 125, 255);
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
           border: 2px solid #335fff;
        }
        QPushButton:hover {
           background-color: rgb(85, 170, 255); 
        }</string>
    </property>
    <property name="text">
     <string>Reservar visita</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(58, 0, 173);</string>
    </property>
    <property name="text">
     <string>Casa da pólvora</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>741</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Conheça o museu mais antigo do Piauí. A Casa da Pólvora é uma construção de uma 
única sala de pedras toscas e rejuntadas com argamassa de barro e representa o grande 
momento militar do Piauí. Construção datada do início do século XIX para abrigar a fábrica 
de pólvora, edificada sobre um único lajedo, às margens do riacho Pouca Vergonha é o 
marco da emancipação do Piauí, durante as lutas pela independência.
</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     /* Cor de fundo vermelha - você pode ajustar conforme necessário */
	
	background-color: rgb(3, 3, 3);
    border: 2px solid #BB0000;  /* Cor da borda */
    color: #FFFFFF;  /* Cor do texto */
    padding: 5px 10px;  /* Espaçamento interno */
    border-radius: 5px;  /* Borda arredondada */
 font-size: 14px;  /* Tamanho da fonte ajustado para 14 pixels */
    font-weight: bold;  /* Fonte em negrito */
}

QPushButton:hover {
    background-color: #CC0000;  /* Cor de fundo ao passar o mouse */
    border: 2px solid #990000;  /* Cor da borda ao passar o mouse */
}
</string>
    </property>
    <property name="text">
     <string>Vídeo </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagens/youtube.png</normaloff>../imagens/youtube.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffffff;  /* Cor de fundo branca */
    border: 2px solid #dddddd;  /* Cor da borda */
    color: #333333;  /* Cor do texto */
    padding: 10px 15px;  /* Espaçamento interno ajustado para um botão maior */
    border-radius: 5px;  /* Borda arredondada */
    font-size: 14px;  /* Tamanho da fonte ajustado para 14 pixels */
    font-weight: bold;  /* Fonte em negrito */
}

QPushButton:hover {
    background-color: #f5f5f5;  /* Cor de fundo ao passar o mouse */
    border: 2px solid #cccccc;  /* Cor da borda ao passar o mouse */
}
</string>
    </property>
    <property name="text">
     <string>Localização</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagens/maps.png</normaloff>../imagens/maps.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
